app/controllers/evidence_controller.rb

```diff
@@ -49,6 +49,11 @@ def create_multiple
     # validate Issue
     issue = current_project.issues.find(evidence_params[:issue_id])
 
+    if node_params_empty?
+      redirect_to project_issue_path(current_project, evidence_params[:issue_id], tab: 'evidence-tab'), alert: 'Evidence cannot be created.'
```

Comment:
We know the error was around Node information, can we provide a better error here?